designs/simplified-scaffolding.md

```diff
@@ -468,6 +469,7 @@ import (
 type MyKindReconciler struct {
 	client.Client
 	log logr.Logger
+	Scheme *runtime.Scheme
```

Comment:
I don't think you should modify this document. Two reasons:
- As said by @mengqiy, as this example doesn't require the `Scheme` adding it just adds compelxity.
- It is a design document, it does not have to reflect future changes.

Reply:
I still think you should not modify the design document.

Reply:
+1